AUTHORS

```diff
@@ -3453,5 +3453,7 @@ Daniel Roe <daniel@roe.dev>
 Niyas Sait <niyas.sait@linaro.org>
 K.C.Ashish Kumar <703559+kcak11@users.noreply.github.com>
 Niklas Mischkulnig <4586894+mischnic@users.noreply.github.com>
+ops+robot@npmjs.com <npm team>
```

Comment:
This looks like a (perhaps intentionally) malformed entry. I would think it should be `npm team <ops+robot@npmjs.com>`. Whatever the case, it's a duplicate entry so we'll want to mailmap it to a single entry.